packages/xs-vat-worker/src/console.js

```diff
@@ -1,7 +1,7 @@
 /** console for xs platform */
 const harden = x => Object.freeze(x, true);
 
-const text = it => (typeof it === 'object' ? JSON.stringify(it) : `${it}`);
+const text = it => (typeof it === 'string' ? it : JSON.stringify(it));
 const combine = (...things) => `${things.map(text).join(' ')}\n`;
 
 export function makeConsole(write_) {
```

Comment:
Curious what the purpose is of naming the parameter `write_` and then assigning it to `write` in the body.

Reply:
I think the code used to say `const write = write_ ? write_ : trace`; i.e. if the caller supplies a `write` capability, use that; otherwise, fall back to ambient authority. Probably should just use `write` now.